pkg/pipeline/transform/transform_network.go

```diff
@@ -213,16 +217,27 @@ func NewTransformNetwork(params config.StageParam) (Transformer, error) {
 		}
 	}
 
+	var servicesDB *netdb.ServiceNames
 	if needToInitNetworkServices {
-		err := netdb.Init(jsonNetworkTransform.ProtocolsFile, jsonNetworkTransform.ServicesFile)
+		var err error
+		protos, err := os.Open("/etc/protocols")
+		if err != nil {
+			return nil, fmt.Errorf("opening /etc/protocols: %w", err)
+		}
+		services, err := os.Open("/etc/services")
```

Comment:
Shouldn't these `os.Open()` be followed by `defer file.Close()` as done in the tests?